manifests/vrrp/instance.pp

```diff
@@ -162,17 +176,20 @@
   $virtual_ipaddress_int                    = undef,
   $virtual_ipaddress_excluded               = undef,
   $virtual_routes                           = undef,
+  $higher_prio_send_advert                  = undef,
```

Comment:
please add datatypes for the new parameters

Reply:
I have added datatype boolean to higher_prio_send_advert and other two has left as it is as they are strings. Thanks

Reply:
If they are strings, please add the string datatype and prohibit empty whitespace:
`String[1]`

Reply:
Sorry, higher_prio_send_advert is an integer. I have added the datatype, and update the template to only update when it defers from default value of 0 as it is an optional parameter. Thanks.